main.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
```

Comment:
Use the standard logger

Reply:
have raised and issue for this https://github.com/ONSdigital/rm-survey-service/issues/17, and its been put on our backlog